mitiq/rem/rem.py

```diff
@@ -87,7 +87,7 @@ def mitigate_executor(
         executor_obj = deepcopy(executor)
 
     def post_run(
-        self: MeasurementResult,
+        self: Executor,
```

Comment:
Is this change intended? It seems unrelated to the other changes, but maybe I'm missing something.

If it is unrelated, happy to keep it in this PR. I imagine you happened upon it while working on this.

Reply:
In fact it's unrelated. It's a small type fix, since `post_run()` is defined to be used as a method of `Executor`s and not of `MeasurementResult`s.  For some reason, this small inconsistency was never detected by `mypy`.